ubuntu/http/preseed.cfg

```diff
@@ -1,6 +1,7 @@
 choose-mirror-bin mirror/http/proxy string
 d-i debian-installer/framebuffer boolean false
 d-i debconf/frontend select noninteractive
+d-i debconf/language select en_HK:en
```

Comment:
I don't want that.

Reply:
Sorry, I'm not sure what I was thinking leaving that in the commit